src/bournemouth/chat_utils.py

```
@@ -58,6 +58,10 @@ class StreamConfig:
     send_lock: asyncio.Lock
     api_key: str
     model: str | None
+    stream_func: typing.Callable[
+        [OpenRouterService, str, list[ChatMessage], typing.Any],
+        typing.AsyncIterator[StreamChunk],
```

Ruff (0.11.9)

Undefined name `StreamChunk`

(F821)